pkg/docker/manager.go

```diff
@@ -167,7 +168,7 @@ func (m *Manager) Watch(ctx context.Context, worker WorkerFn, labels ...string)
 			handle := m.NewContainerRef(containerID)
 			err := worker(cctx, handle)
 			if err != nil {
-				if errors.Is(err, context.Canceled) {
+				if errors.Is(err, context.Canceled) || strings.Contains(err.Error(), "context canceled") { // docker doesn't wrap errors
```

Comment:
Because of:

```
Apr 28 19:54:41.924868	[31mERROR[0m	sidecar worker failed: failed to initialise the container: inspect failed: error during connect: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.40/containers/96b9615ab3ab68667249a32a61f3887123a6461f61874e9089943e4713452157/json": context canceled	{"host": "unix:///var/run/docker.sock", "container": "96b9615ab3ab68667249a32a61f3887123a6461f61874e9089943e4713452157"}
```